File: 1-two-sum/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        diff = []
        for i in range(0, len(nums)):
            if target - nums[i] in diff:
                return [i, diff.index(target - nums[i])]
            diff.append(nums[i])
        

File: 1-two-sum/two-sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        HashMap<Integer, Integer> seen = new HashMap<>();
        for (int i = 0; i < nums.length; i++) {
            if (seen.containsKey(nums[i])) {
                return new int[]{i, seen.get(nums[i])};
            } else {
                seen.put(target - nums[i], i);
            }
        }
        return new int[]{0, 0};
    }
}